### ISSUE TITLE
Microphone: Effects/Filters selects do not have labels

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#243]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/microphone/

Might want to think about this as all the labels are H3's, but the first two are not the same as filter and effects (the latter are true labels for their controls, but the first two are a label for a bunch of controls).

If we don't want to change "effects" and "filters" to a label element, it should have an ID added and the select should have an aria-labelledby attribute pointing to that ID.
